src/encoded/tests/test_audit_biosample_type.py

```diff
@@ -2,23 +2,50 @@
 
```

Comment:
I feel like you should add a test for a "clean" biosample that returns no audits.  Also, I think it's somewhat cleaner (although more tedious & verbose) to break def test_audit_biosample_type_term into 1 test per error.   When you do fail tests it will be more clear what part of the code is broken.

Reply:
Great suggestion! Updated in the new commit.